### ISSUE TITLE
docker-compose linked from above directory still not showing up in solution tree view

### ISSUE THREAD

**Issue by ggirard07** [microsoft/DockerTools#367]
#336 has been closed as resolved since v17.4 [according to comment](https://github.com/microsoft/DockerTools/issues/336#issuecomment-1397703384).
But v17.4.4 user here and still seeing this exact issue where linked file are not showing up.

`.dcproj` content
```xml
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="15.0" Sdk="Microsoft.Docker.Sdk">
  <PropertyGroup Label="Globals">
    <ProjectVersion>2.1</ProjectVersion>
    <DockerTargetOS>Linux</DockerTargetOS>
    <ProjectGuid>...</ProjectGuid>
    <DockerComposeBaseFilePath>../docker-compose</DockerComposeBaseFilePath>
    <DockerComposeProjectName>...</DockerComposeProjectName>
  </PropertyGroup>
  <ItemGroup>
    <!-- Project items in relative directories do not currently show up in soluion explorer -->
    <!-- https://github.com/microsoft/DockerTools/issues/336 -->
    <None Include="../docker-compose.yml" />
    <None Include="../.dockerignore" />
  </ItemGroup>
</Project>
```

Solution tree
![image](https://user-images.githubusercontent.com/2625886/213716669-4b47b94c-18d0-41d8-a182-0453bae07c37.png)

**Comment by NCarlsonMSFT** [comment]
@ggirard07 I played with this a little bit and can confirm that the Items you have above aren't showing for me, you can get them to show with:
```xml
  <ItemGroup>
    <None Include="../docker-compose.yml">
      <Link>docker-compose.yml</Link>
    </None>
    <None Include="../.dockerignore">
      <Link>.dockerignore</Link>
    </None>
  </ItemGroup>
```

**Comment by ggirard07** [comment]
@NCarlsonMSFT nice, can confirm workaround is working for me too.
Out of curiosity, do you know out of hand whether the workaround should for version prior v17.4 too?

**Comment by NCarlsonMSFT** [comment]
@patverb can you confirm that this is only working in 17.4+?

**Comment by patverb** [comment]
@ggirard07 @NCarlsonMSFT Looks like I got that fix in for the 17.3 build so it should actually be 17.3+

**Comment by dbreshears** [comment]
Since there is a workaround for this, going to close this as lower priority.
